resources/js/views/login.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="container">
+        <h1 class="text-center login-title">iSTART</h1>
```

Comment:
dis ain't BEM 😢

Reply:
in the container class make `.login` and then in the text-center class to `.login__title`

Reply:
thank